projects/plugins/protect/src/js/components/threats-list/index.jsx

```diff
@@ -61,15 +73,20 @@ const ThreatsList = () => {
 							<Title className={ styles[ 'list-title' ] }>{ getTitle() }</Title>
 							{ hasRequiredPlan && (
 								<>
-									{ fixableCount > 0 && (
+									{ fixableList.length > 0 && (
 										<Button variant="primary" className={ styles[ 'list-header-button' ] }>
 											{
 												/* translators: Translates to Auto fix all. $s: Number of fixable threats. */
-												sprintf( __( 'Auto fix all (%s)', 'jetpack-protect' ), fixableCount )
+												sprintf( __( 'Auto fix all (%s)', 'jetpack-protect' ), fixableList.length )
 											}
 										</Button>
 									) }
-									<Button variant="secondary" className={ styles[ 'list-header-button' ] }>
+									<Button
+										variant="secondary"
+										className={ styles[ 'list-header-button' ] }
+										isLoading={ Boolean( scanIsEnqueuing && scanIsEnqueuing === true ) }
```

Comment:
```suggestion
										isLoading={ scanIsEnqueuing }
```
Since `scanIsEnqueuing` should already be a boolean, I don't think we need to be so verbose with the check. If you want a strict boolean check, just `scanIsEnqueuing === true` will work as well!